intellij-plugin/build.gradle.kts

```diff
@@ -135,6 +136,10 @@ tasks.configureEach {
   }
 }
 
+val apolloPublished = configurations.dependencyScope("apolloPublished").get()
+
+configurations.getByName("implementation").extendsFrom(apolloPublished)
```

Comment:
This is so that I could do `includeBuild("../apollo-mockserver")` (see also the build logic part with `globalSubstitutionRules`). Without this the included build doesn't get resolved.